packages/@vuepress/plugin-google-analytics/src/client/composables/useGoogleAnalytics.ts

```diff
@@ -41,5 +45,10 @@ export const useGoogleAnalytics = (id: string): void => {
   }
 
   gtag('js', new Date())
-  gtag('config', id)
+
+  if (options.debug) {
+    gtag('config', options.id, { debug_mode: true })
+  } else {
+    gtag('config', options.id)
+  }
```

Comment:
Can we use `gtag('config', options.id, { debug_mode: options.debug === true })` ? I'm not sure if setting `{ debug_mode: false }` will cause any different behavior

Reply:
That would make sense and look nice, but unfortunately they said this on the page I linked in the description:

> To disable debug mode, exclude the 'debug_mode' parameter; setting the parameter to 
 false doesn't disable debug mode.